cmd/conveyor/worker.go

```diff
@@ -30,11 +30,23 @@ var workerFlags = []cli.Flag{
 		EnvVar: "DRY",
 	},
 	cli.StringFlag{
+		Name:   "builder",
+		Value:  "conveyor",
+		Usage:  "The builder backend to use. Can be \"codebuild\" or \"docker\"",
+		EnvVar: "BUILDER",
+	},
+	cli.StringFlag{
 		Name:   "builder.image",
 		Value:  docker.DefaultBuilderImage,
-		Usage:  "A docker image to use to perform the build.",
+		Usage:  "A docker image to use to perform the build. Only used with the Docker backend.",
```

Comment:
Yeah, the default already gets included. The help text for this option looks like:

```
--builder.image 'remind101/conveyor-builder:master'  A docker image to use to perform the build. Only used with the Docker backend. [$BUILDER_IMAGE]
```